src/errors.rs

```diff
@@ -48,12 +47,19 @@ quick_error! {
         ConfigInvalid(desc: String) {
             description(desc)
         }
-        /// A Protobuf message failed in some manner.
-        Codec(err: ProtobufError) {
```

Comment:
I think we still need this error since we can still through protobuf errors via the protobuf Message trait functions

Reply:
But currently I didn't see a compilation error related to the protobuf error. Is it even possible to have such error while compiler is not aware of that?

Reply:
I get an error in TiKV - it's possible in the same way that if you remove a public fn which isn't used internally, then you won't see an error until it is used downstream